lib/pages/people/view/person_view.dart

```diff
@@ -1,104 +1,19 @@
 import 'package:acs_upb_mobile/pages/people/model/person.dart';
 import 'package:acs_upb_mobile/pages/people/service/person_provider.dart';
-import 'package:acs_upb_mobile/resources/utils.dart';
-import 'package:acs_upb_mobile/widgets/icon_text.dart';
-import 'package:cached_network_image/cached_network_image.dart';
+import 'package:acs_upb_mobile/pages/people/view/courses_list_view.dart';
+import 'package:acs_upb_mobile/pages/people/view/person_banner.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_feather_icons/flutter_feather_icons.dart';
 import 'package:provider/provider.dart';
 
-class PersonView extends StatelessWidget {
-  const PersonView({Key key, this.person}) : super(key: key);
+import 'contact_info.dart';
 
+class PersonView extends StatefulWidget {
+  const PersonView({Key key, this.person}) : super(key: key);
   final Person person;
 
   @override
-  Widget build(BuildContext context) {
```

Comment:
Woah there, you can't just replace the entire code here. We need it for mobile.

Reply:
Upss, I solved it [here](https://github.com/student-hub/acs-upb-mobile/blob/396dff90559907416f9ea31238fb87d37c747e59/lib/pages/people/view/person_view.dart#L48-L180)